src/ViewHoldingsRecord.js

```diff
@@ -220,6 +228,8 @@ class ViewHoldingsRecord extends React.Component {
     if (holdings.permanentLocationId === '') delete holdings.permanentLocationId;
     if (holdings.temporaryLocationId === '') delete holdings.temporaryLocationId;
 
+    this.setState({ isLoadingUpdatedHoldingsRecord: true });
```

Comment:
@BogdanDenis would you mind explaining why we need this extra local state var?
The `accumulate` should still trigger `ACC_FETCH_SUCCESS` which flips `hasLoaded` and `isPending`. 

https://github.com/folio-org/stripes-connect/blob/9c53099c9bf09bf83723b988b0d07d8e52202fac/RESTResource/reducer.js#L41-L43

Could you use these flags instead?

Reply:
@mkuklis Maybe I'm misunderstanding something, but `hasLoaded` is only `false` in initial state and is not reset after that. And I'm worried that changing it is probably going to break a lot of things in a lot of modules
But we still need this state variable because we need to know that Holdings record is loading because of an update or not. If we're loading after update - we need to wait for two requests to finish. Otherwise delay between PUT and GET requests will cause Instance record to appear

Reply:
@BogdanDenis if it's not reset after that then it's a bug in stripes-connect we should address. You can also use `isPending` which seems to be working as expected.

Reply:
@mkuklis `isPending` is working correctly when it's just one request, but when we have two requests (for update and for refresh) - there's a slight delay between them when `isPending` is false. So we need a flag that stays `true` throughout the whole time it takes to update and refresh the record.
Also, `isPending` is not enough to differentiate between loading a record when it's opened to view and loading it when it's been updated and needs to be refreshed.

Reply:
@BogdanDenis sounds good. To me that local state is just a hack to go around the bug in stripes-connect. I understand that you want to move forward with this. Perhaps we can add a little comment to that code for now why this local state is required and go ahead and merge. I wonder if instead of using stripes-connect in the future we should try to use react-query.